test/e2e/wmco_test.go

```diff
@@ -25,6 +26,17 @@ var (
 	cleanupTimeout       = time.Second * 5
 )
 
+const (
+	// api version of WindowsMachineConfig custom resource
+	wmcApiVersion = "wmc.openshift.io/v1alpha1"
+	// account ID of AWS account
+	awsAccountID = "default"
+	// sshKey is the key that will be used to access created Windows VMs
+	sshKey = "libra"
```

Comment:
unnecessary changes in this PR

Reply:
They were getting repeated at too many places so I thought of making them constants

Reply:
I am a +1 on this given he would have to copy paste them otherwise. Alternatively making the constants could have been a separate commit.

Reply:
Lets handle this when we rebase